### DISCUSSION TITLE
Validation Schema availability JSON/XSD

### DISCUSSION THREAD

**Discussion by bobstuart** [ocsf/ocsf-schema#553] [Q&A] [answered]
Are there JSON Schemas available that are generated from the OCSF Schema? 
Or XSD Schemas. Looking for how to validate that data at some level conforms with OCSF without hand writing all the validation myself. 

Apologies in advance if they are right in front of my face,  I did look but didn't see it.

**Comment by rroupski** [comment, marked as answer]
The schema server has an API to validate data and to export JSON schema per class or object, see the [API doc](https://schema.ocsf.io/doc). For example, to export the network activity event class use:

`curl -X GET "https://schema.ocsf.io/schema/classes/network_activity" -H "accept: application/json"`
